src/Libraries/CoreNodes/CoreNodes.csproj

```diff
@@ -40,6 +40,11 @@
       <Project>{c0d6dee5-5532-4345-9c66-4c00d7fdb8be}</Project>
       <Name>DesignScriptBuiltin</Name>
     </ProjectReference>
+    <ProjectReference Include="..\DynamoUnits\UnitsCore.csproj">
```

Comment:
why is this necessary? I recall a lot of work untangling these dependencies.
I have not delved back in but this change might necessitate taking another look at the nuget packages.

Reply:
In terms of Nuget I think we are good.  CoreNodes (ie DSCoreNode.dll) is in the DynamoVisualProgramming.DynamoCoreNodes nuget.  That nuget depends on DynamoVisualProgramming.ZeroTouchLibrary nuget which includes DynamoUnits.dll aka UnitsCore.csproj.  That being said the purpose of this is to make it so we can serialize `Dynamo.Location` objects.  Not sure how important that one is